spec/sales_engine_spec.rb

```diff
@@ -39,7 +50,37 @@
       mr = se.merchants
 
       expect(mr).to be_an_instance_of(MerchantsRepository)
+=======
+      expect(se.merchants).to be_an_instance_of(MerchantsRepository)
+      expect(se.items).to be_an_instance_of(ItemRepository)
+>>>>>>> 0705cc0ff1d05bf152bded540317d0d30835850b
+    end
+  end
+
+  describe '#analyst' do
+<<<<<<< HEAD
+    it 'creates a new instance of SalesAnalyst' do
+      se = SalesEngine.new(
+        :items     => "./data/items.csv",
+        :merchants => "./data/merchants.csv"
+        )
+
+      analyst = se.analyst(se.items, se.merchants)
+
+      expect(analyst).to be_a(SalesAnalyst)
+    end
+  end
+=======
+    it 'can create an instance of SalesAnalyst' do
+      se = SalesEngine.new({
+        :items     => "./data/items.csv",
+        :merchants => "./data/merchants.csv"
+        })
+      sales_analyst = se.analyst(se.items, se.merchants)
+
+      expect(sales_analyst).to be_an_instance_of(SalesAnalyst)
     end
   end
 
+>>>>>>> 0705cc0ff1d05bf152bded540317d0d30835850b
```

Comment:
Noticing a lot of these remnants of merging; the lines that start with '>>>>>>', '======', and '<<<<<<'. Please get rid of these, as they can mess with overall functionality and in general just further clutter the code.